components/sup/src/manager/service/hooks.rs

```diff
@@ -20,6 +19,25 @@ use std::{self,
 
 static LOGKEY: &str = "HK";
 
+#[derive(Debug)]
+pub struct CompleteHookOutput {
```

Comment:
Having types named `HookOutput` and `CompleteHookOutput` running around together is a little confusing to me... I wonder if there are alternative names or type structures we could use that would make the distinction between these, as well as their intended uses, more clear.